server/repository/src/db_diesel/changelog/README.md

```diff
@@ -0,0 +1,46 @@
+# Changelog
+
+A number of operations rely on processing a `queue` of database changes.
+These operations include pushing records to central server during synchronisation, processing transfer records, processing messaging, aggregation, interoperability synchronisation, etc.. 
+
+Generic mechanism for recording and querying these database changes was implemented in the form of the `changelog` table.
+
+## Triggers
+
+Database triggers insert new entries into changelog table with:
+
+* cursor <- sequential number for order of database operation
+* record_id 
+* table_name
+* store_id <- the store that `owns` the record
+* name_id <- the name_id of the `destination` entity (can also be a store)
+
+## changelog_deduped view
+
+Current use cases require only the latest operation for a particular record, thus a database view is used to query latest operation grouped by unique record_id.
+
+`deduping` is short for deduplication (process of removing identical entries)
+
+## Cursors
+
+The 'cursor' is used by changelog consumers to track which records they have already processed and allow them to start processing again from where they left off.
```

Comment:
the 'quotes' aren't required - it is a cursor that we're talking about

Reply:
Thanks, it's from a suggestion btw: https://github.com/openmsupply/open-msupply/pull/571#discussion_r967939795, removed 256869b3c4be2e510ec52b029561f7f25d32771f